lib/webtask.js

```diff
@@ -708,6 +756,42 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function validate_auth_token(secret, expected_aud) {
+            var auth_token;
+            var decoded_auth_token;
+            var userinfo;
+
+            console.log('\n\n\n\n');
+            console.log(secret);
+
+            try {
+                secret = new Buffer(secret, 'base64');
+            } catch(e) {
+                throw { code: 403, message: 'WEBTASK_JWT_SECRET must be base64 encoded: ' + e.message, req_id: req.req_id };
```

Comment:
We should move this check to the point of webtask token issuance (the `validate_token_issue_request`) rather than performing it for every webtask request.